cmd/snap/cmd_aliases.go

```diff
@@ -45,8 +45,8 @@ $ snap aliases <snap>
 Lists only the aliases defined by the specified snap.
 
 An alias noted as undefined means it was explicitly enabled or disabled but is
-not defined in the current revision of the snap; possibly temporarely (e.g
```

Comment:
:heart:

Reply:
temporarely -- Something that happens for a short period of time, and only rarely.